root/static/scripts/common/components/Autocomplete2/recentItems.js

```diff
@@ -179,6 +186,7 @@ export async function getOrFetchRecentItems<+T: EntityItemT>(
 ): Promise<$ReadOnlyArray<OptionItemT<T>>> {
   const ids = _getRecentEntityIds(key);
   const cachedList: Array<OptionItemT<T>> = [...getRecentItems<T>(key)];
+  const isLanguageForWorks = key === 'language-lyrics';
```

Comment:
I was gonna suggest passing this to the other `getEntityName` call too, but I doubt there will ever be a case where the languages aren't loaded in `linkedEntities`.

Can we move this under `if (ids.size) {` then, since it's not used outside of that scope? (It's a tiny thing, but can be annoying when there are irrelevant variables in scope when using the step debugger.)